src/maga/character/Trump.java

```diff
@@ -14,25 +15,60 @@
  */
 public class Trump extends Character {
     /**
+     * Integer that sets how many rounds Trump should wait.
+     */
+    private int roundsToWait;
```

Comment:
This should be initialized with 0

Reply:
fixed in: 871f79b749c3891c94e586cfcfecd9376b71ed2c